Sources/XCRemoteCache/Config/XCRemoteCacheConfig.swift

```diff
@@ -129,6 +129,8 @@ public struct XCRemoteCacheConfig: Encodable {
     /// `/CoolLibrary/main.swift`will be represented as `$(COOL_LIBRARY)/main.swift`).
     /// Warning: remapping order is not-deterministic so avoid remappings with multiple matchings.
     var outOfBandMappings: [String: String] = [:]
+    /// Option to enable/disable TrustManager
+    var certificateVerification: Bool = true
```

Comment:
I looked on other tools and found that  many of them use a negation of that Bool. Examples:
* Chrome: `--ignore-certificate-errors`
* git: `GIT_SSL_NO_VERIFY`
* wget: `--no-check-certificate`
* 
Maybe we can have something like:
```suggestion
    var disableCertificateVerification: Bool = false
```